rust/noosphere-core/src/tracing.rs

```diff
@@ -305,23 +309,45 @@ mod inner {
 
         let subscriber = tracing_subscriber::registry().with(env_filter);
 
+        // This isn't a secret value, just where to send the error reports. We can make this configurable in the future if other folks care.
+        let _guard = sentry::init(("https://553fd6eda33842ed9f088d0c16a147f1@o4505393671569408.ingest.sentry.io/4505399702126593", sentry::ClientOptions {
```

Comment:
We should only initialize Sentry if the feature flag is set (otherwise we'll get an error when it is not set, because the related dependencies won't be available).